app/helpers/automatic/process_pool.py

```
@@ -0,0 +1,619 @@
+                stdin=asyncio.subprocess.PIPE,
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.STDOUT,  # Merge stderr for easier logging
+                env=os.environ,
+            )
+
+            voice_process = VoiceAgentProcess(proc, process_id, is_managed=True)
+            self.managed_processes[process_id] = voice_process
+
+            await self._wait_for_process_ready(voice_process)
+
+            asyncio.create_task(self._monitor_process_output(voice_process))
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)